### ISSUE TITLE
[Keyboard Navigation – Make Code - Review, Commit and push local changes to GitHub ]Controls like: 'MASTER' present on screen' is not accessible using keyboard.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3762]
User Experience:
If 'any control will not be accessible through keyboard then users who are dependent on keyboard for navigation will not be able to access that control and perform actions which are associated to that control.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Review, Commit and push local changes to GitHub" button and select it.
6. Navigate through all the controls appearing.
7. Verify the issue

Actual Result:
controls like: 'MATER' is not accessible using keyboard.

Expected Result:
All the interactive element should be accessible using keyboard as well as mouse.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[MAS 2.1.1 - Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG2.1.1; #Win10-Edge(Chromium);

![Uploading WCAG2.1.1.png…]()
